### ISSUE TITLE
Can't set SSL verify mode in faraday options

### ISSUE THREAD

**Issue by taquitos** [octokit/octokit.rb#981]
When people use proxies for connection, they might want to disable SSL verification. This often helps during debugging when using a proxy like [Charles Proxy](https://www.charlesproxy.com).

The primary driver for this feature is to enable developers to use a proxy while they are debugging their own application that makes use of octokit.

**Comment by taquitos** [comment]
Here's an example of usage:
```
if ENV['PROXY']
  Octokit.configure do |c|
    # for debugging only
    # This enables tracking of networking requests using Charles Web Proxy
    c.proxy = "https://127.0.0.1:8888"
    c.ssl_verify_mode = OpenSSL::SSL::VERIFY_NONE
  end        
end
```

You can see how we do something very similar in _fastlane_ [client.rb usage of verify mode](https://github.com/fastlane/fastlane/blob/1a79353a6d918eddca61274e97962a5f2770210e/spaceship/lib/spaceship/client.rb#L220-L224)

**Comment by BasThomas** [comment]
Think this can be closed as #982 has been merged. :)

**Comment by taquitos** [comment]
Sorry about that, thank you! 🎈:goat:
